### PR TITLE
Fixes #133 row_index_table can't handle empty dataframes

### LINKED ISSUE #133: notebooks.row_index_table can't handle empty dataframes
I have a slicer that returns an empty dataframe (as expected) when called with 

```
my_slicer.manager.data()
```

but when called with 

```
my_slicer.notebooks.row_index_table()
```

it throws an Exception as follows:

```
(some parts of the stacktrace removed)

/Users/aweller/.virtualenvs/antfarm-strategies34/lib/python3.4/site-packages/fireant/slicer/managers.py in _get_and_transform_data(self, tx, metrics, dimensions, metric_filters, dimension_filters, references, operations)
    464         display_schema = self.manager.display_schema(metrics, dimensions, references, operations)
    465 
--> 466         return tx.transform(dataframe, display_schema)

/Users/aweller/.virtualenvs/antfarm-strategies34/lib/python3.4/site-packages/fireant/slicer/transformers/notebooks.py in transform(self, dataframe, display_schema)
     15 class PandasRowIndexTransformer(Transformer):
     16     def transform(self, dataframe, display_schema):
---> 17         dataframe = self._set_display_options(dataframe, display_schema)
     18         if display_schema['dimensions']:
     19             dataframe = self._set_dimension_labels(dataframe, display_schema['dimensions'])

/Users/aweller/.virtualenvs/antfarm-strategies34/lib/python3.4/site-packages/fireant/slicer/transformers/notebooks.py in _set_display_options(self, dataframe, display_schema)
     37 
     38                 if isinstance(dataframe.index, pd.MultiIndex):
---> 39                     dataframe.index.set_levels(display_values, key, inplace=True)
     40 
     41                 else:

/Users/aweller/.virtualenvs/antfarm-strategies34/lib/python3.4/site-packages/pandas/indexes/multi.py in set_levels(self, levels, level, inplace, verify_integrity)
    236             if not is_list_like(levels):
    237                 raise TypeError("Levels must be list-like")
--> 238             if is_list_like(levels[0]):
    239                 raise TypeError("Levels must be list-like")
    240             level = [level]

IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by mikeengland**

### CODE REVIEW

File: fireant/tests/mock_dataframes.py [hunk 1]
```diff
@@ -60,10 +60,10 @@ def rollup(dataframe, levels):
 
 # Mock DF with single metric column and no dimension
 no_dims_single_metric_df = pd.DataFrame(
-    np.array([
-        np.arange(1),
-    ]),
-    columns=['one'],
+        np.array([
```
Comment (twheys) [inline fireant/tests/mock_dataframes.py]: are all these added tabs necessary?  We have been using 4 spaces for indentation every where else, let's keep it at that please.

Comment (mikeengland) [reply fireant/tests/mock_dataframes.py]: This is PyCharm's default formatter.

Comment (twheys) [reply fireant/tests/mock_dataframes.py]: Should be 4 spaces, pep8 https://www.python.org/dev/peps/pep-0008/#indentation

Comment (mikeengland) [reply fireant/tests/mock_dataframes.py]: Not sure what caused it but I just pushed a fix.

**Pull request merged**